response/slack/client.py

```diff
@@ -83,6 +83,25 @@ def get_channel_id(self, name):
 
         raise SlackError(f"Channel '{name}' not found")
 
+    def get_usergroup_id(self, group_name):
+        response = self.api_call("usergroups.list")
+        if not response.get("ok", False):
+            raise SlackError('Failed to get usergroup with group name {} : {}'.format(group_name, response['error']))
```

Comment:
Minor style point but we could use python 3 `f""` format strings here (I think we try and use them across this codebase).

Reply:
Ah, that was copy+pasted from the old code.  I'll update 👍